test/fixtures/businesses/hiring_invoice_requests.yml

```
@@ -0,0 +1,9 @@
+one:
+  business_id: <%= ActiveRecord::FixtureSet.identify(:one) %>
+  billing_address: 123 Sample Business St.
+  developer_name: John Doe
+  position: Rails Devleoper
+  start_date: <%= Date.current.tomorrow %>
```

```suggestion
  start_date: <%= Date.current + 7.days %>
```

It's a tiny chance but possible to cross over midnight during a test run.